### ISSUE TITLE
Warning about deprecated stage names in `pre-commit-hooks`

### ISSUE THREAD

**Issue by mushitoriami** [sphinx-contrib/sphinx-lint#147]
When running `tox`, the following warning appears:

```
[WARNING] repo `https://github.com/pre-commit/pre-commit-hooks` uses deprecated stage names (commit, push) which will be removed in a future version.  Hint: often `pre-commit autoupdate --repo https://github.com/pre-commit/pre-commit-hooks` will fix this.  if it does not -- consider reporting an issue to that repo.
```

It seems that this warning is caused by an outdated version of `pre-commit-hooks`.
As suggested in the message, updating `pre-commit-hooks` with:

```
pre-commit autoupdate --repo https://github.com/pre-commit/pre-commit-hooks
```

resolves the warning.

It also occurs in GitHub Actions workflows, for example here: https://github.com/sphinx-contrib/sphinx-lint/actions/runs/18796404715/job/53646477582?pr=146#step:4:119
